002-quarkus-all-extensions/src/main/resources/application.properties

```diff
@@ -18,14 +18,12 @@ quarkus.kafka-streams.topics=kafka-topic
 
 quarkus.artemis.url=foo
 
-# Disabled check pom.xml for more info
-#quarkus.oauth2.client-id=client_id
-#quarkus.oauth2.client-secret=secret
-#quarkus.oauth2.introspection-url=http://oauth-server/introspect
-
-# Disabled needs running keycloak server
-#quarkus.oidc.auth-server-url=https://localhost:8180/auth/realms/quarkus
-#quarkus.oidc.client-id=quarkus-app
+quarkus.oauth2.client-id=client_id
+quarkus.oauth2.client-secret=secret
+quarkus.oauth2.introspection-url=http://oauth-server/introspect
```

Comment:
I have doubt that these 3 properties has any effect at all. Do they?

Reply:
Yes, they are required by `quarkus-elytron-security-oauth2` extension. 
https://quarkus.io/guides/security-oauth2#setting-up-application-properties

Reply:
sorry, that's not the answer, `quarkus.oauth2.introspection-url=http://oauth-server/introspect` is invalid, so what that it is required as long as it is not as much as validated or used at all?

Reply:
Anyway, I'll close this comment as this extension is clearly not used at all. There is no point of talking about it.

Reply:
That's the part of minimal config according to docs and it caused exception `Caused by: io.quarkus.runtime.configuration.ConfigurationException: client-id, client-secret and introspection-url must be configured when the oauth2 extension is enabled
`

Reply:
> That's the part of minimal config according to docs and it caused exception `Caused by: io.quarkus.runtime.configuration.ConfigurationException: client-id, client-secret and introspection-url must be configured when the oauth2 extension is enabled `

I am not sure how to state it differently, the introspection URL is clearly invalid, hence it is clearly not used or validated. I don't care about required config property. The point here is, it brings no value to use this apart of having this extension present but not used.

Let's not continue.

Reply:
Of course, keep the property. I was just commenting on the point that extension is not used.